Cargo.toml

```diff
@@ -36,11 +36,17 @@ tempfile = "3.8"
 arrow2 = "0.18"
 
 [workspace]
-members = ["gdal-sys"]
+members = ["gdal-src", "gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+proj-sys = { git = "https://github.com/GiGainfosystems/proj", rev = "dc67b5afc17bc7d330371e4406dffd65cf9321cd" }
+libsqlite3-sys = { git = "https://github.com/rusqlite/rusqlite/", rev = "42a82fb" }
+pq-sys = { git = "https://github.com/sgrif/pq-sys", rev = "77e6729" }
```

Comment:
also here, would it be possible to link to some fixed version rather then some revisions in the GitHub repos?

Reply:
That's certainly possible as soon as those patches are merged in these repositories and fixed versions are released. For `libsqlite3-sys` that's already the case, for `pq-sys` I need to cut an release, for `proj-sys` I need to wait for the maintainers there to decide what to do.

Reply:
Patches for pq-sys and libsqlite3-sys are removed as the relevant releases are published to crates.io. For proj-sys is still wait for feedback from the maintainers.